src/PageLink.vue

```diff
@@ -0,0 +1,27 @@
+<script setup lang="ts">
+import type { PageLinkProps } from './objects';
+import { computed } from 'vue';
+
+const props = defineProps<PageLinkProps>();
+
+const title = computed(() => {
+  const vowels = 'aeiou';
+  const firstLetter = props.text.slice(0, 1).toLowerCase();
+  return `Compress ${vowels.includes(firstLetter) ? 'a' : 'an'} photo in ${props.text}`;
+});
```

Comment:
It's always "a photo"
```suggestion
const title = `Compress a photo in ${props.text}`;
```